ansible/roles/ocp4-workload-integreatly/templates/clusterrole.j2

```diff
@@ -0,0 +1,52 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ minio_operator_role }}
```

Comment:
And of course here you'd use `{{ ocp4-workload-integreatly.minio_operator_role }}`